src/cli/commands/lint.ts

```diff
@@ -11,6 +11,7 @@ import { json, stylish } from '../../formatters';
 import { readParsable } from '../../fs/reader';
 import { httpAndFileResolver } from '../../resolvers/http-and-file';
 import { getDefaultRulesetFile } from '../../rulesets/loader';
+import { isOAS2Spec, isOAS3Spec } from '../../rulesets/lookups';
```

Comment:
Some terminology style guide stuff here, instead of saying "Spec" (JSON is a spec, YAML is a spec, HTTP is a spec, we dont say spec after all of them), and the S in OAS is already "spec" anyway.

API Descriptions are documents written in special formats, so there is a description Document - the file(s) - and description Formats (OpenAPI, RAML, etc). 

Can we try and make sure that at least our new code is consistent with this style guide, and maybe boy scout anything internal we bump into on the way?

tldr: `isOpenApiV2` & `isOpenApiV3`

Reply:
oh we should probably make them isOpenApiv2_0, isOpenApiv3_0, isOpenApiv3_1, etc or something less ugly. People will want to have special rules for 3.0 like "hey upgrade to 3.1" lol

Reply:
Good catch. Didn't pay much attention to that, but you are absolutely right about it.
Will do.